### PR TITLE
Marketplace fixes

### PR DESCRIPTION
**Pull request by JulianaMeyerArruda**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

**Quick fixes to the Marketplace API!**
Includes improved descriptions of some fields, and correcting object's types. 
Part of the following issues:
- [EDU-3786](https://vtex-dev.atlassian.net/browse/EDU-3786)
- [EDU-3626](https://vtex-dev.atlassian.net/browse/EDU-3626)
- [EDU-3201](https://vtex-dev.atlassian.net/browse/EDU-3201)

### CODE REVIEW

File: VTEX - Marketplace APIs.json [hunk 1]
```diff
@@ -687,7 +699,7 @@
                 {
                     "name": "matcherid",
                     "in": "query",
-                    "description": "Identifies the matching entity. It can be either VTEX's matcher, or an external matcher developed by partners, for example.",
+                    "description": "Identifies the matching entity. It can be either VTEX's matcher, or an external matcher developed by partners, for example. The `matcherId`'s value can be obtained through the *Get SKU Suggestion by ID* endpoint.",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:702]: Acho que podia adicionar o link do endpoint citado aqui

File: VTEX - Marketplace APIs.json [hunk 2]
```diff
@@ -1244,7 +1256,7 @@
                 {
                     "name": "version",
                     "in": "path",
-                    "description": "Whenever an SKU Suggestion is updated or changed, a new version of the original one is created. All versions are logged, so you can search for previous our current states of SKU suggestions. This field is the versionId associated to the version you choose to search for.",
+                    "description": "Whenever an SKU Suggestion is updated or changed, a new version of the original one is created. All versions are logged, so you can search for previous our current states of SKU suggestions. This field is the versionId associated to the version you choose to search for. You can get this field's value through the *Get SKU Suggestion by ID*. through the `latestVersionId` field.",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:1259]: Mesma coisa aqui, adicionar o link do endpoint

File: VTEX - Marketplace APIs.json [hunk 3]
```diff
@@ -1254,7 +1266,7 @@
                 {
                     "name": "matchid",
                     "in": "path",
-                    "description": "Whenever an SKU suggestion is matched, it is associated to a unique ID. Fill in this field with the matchId you wish to filter by.",
+                    "description": "Whenever an SKU suggestion is matched, it is associated to a unique ID. Fill in this field with the matchId you wish to filter by. The `matchId`'s value can be obtained through the *Get SKU Suggestion by ID* endpoint.",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:1269]: Mesma coisa aqui, adicionar o link do endpoint

File: VTEX - Marketplace APIs.json [hunk 4]
```diff
@@ -1809,7 +1823,7 @@
             "properties": {
               "MatcherId": {
                 "type": "string",
-                "description": "Identifies the matching entity. It can be either VTEX's matcher, or an external matcher developed by partners, for example.",
+                "description": "Identifies the matching entity. It can be either VTEX's matcher, or an external matcher developed by partners, for example. The `matcherId`'s value can be obtained through the *Get SKU Suggestion by ID* endpoint.",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:1826]: Mesma coisa aqui, adicionar o link do endpoint

File: VTEX - Marketplace APIs.json [hunk 5]
```diff
@@ -2145,7 +2151,7 @@
             "properties": {
                 "matcherId": {
                     "type": "string",
-                    "description": "Identifies the matching entity. It can be either VTEX's matcher, or an external matcher developed by partners, for example.",
+                    "description": "Identifies the matching entity. It can be either VTEX's matcher, or an external matcher developed by partners, for example. The `matcherId`'s value can be obtained through the *Get SKU Suggestion by ID* endpoint.",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:2154]: Mesma coisa aqui, adicionar o link do endpoint

File: VTEX - Marketplace APIs.json [hunk 6]
```diff
@@ -2383,70 +2389,75 @@
           },
           "MatchMultiple": {
             "title": "MatchMultiple",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:2391]: Colocar `title` como `""` para tirar esse nome da visualização da documentação

File: VTEX - Marketplace APIs.json [hunk 7]
```diff
@@ -2383,70 +2389,75 @@
...
+                            "default": ""
+                        },
+                        "versionId": {
+                            "type": "string",
+                            "description": "Whenever an SKU Suggestion is updated or changed, a new version of the original one is created. All versions are logged, so you can search for previous our current states of SKU suggestions. This field is the versionId associated to the version you choose to search for. You can get this field's value through the *Get SKU Suggestion by ID*. through the `latestVersionId` field.",
+                            "default": ""
+                        },
+                        "matchId": {
+                            "type": "string",
+                            "description": "Whenever an SKU suggestion is matched, it is associated to a unique ID. Fill in this field with the matchId you wish to filter by. The `matchId`'s value can be obtained through the *Get SKU Suggestion by ID* endpoint.",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:2418]: Mesma coisa aqui, adicionar o link do endpoint

File: VTEX - Marketplace APIs.json [hunk 8]
```diff
@@ -2383,70 +2389,75 @@
...
+                            "default": ""
+                        },
+                        "matcherId": {
+                            "type": "string",
+                            "description": "Identifies the matching entity. It can be either VTEX's matcher, or an external matcher developed by partners, for example. The `matcherId`'s value can be obtained through the *Get SKU Suggestion by ID* endpoint.",
```
Comment (karenkrieger) [inline VTEX - Marketplace APIs.json:2423]: Mesma coisa aqui, adicionar o link do endpoint

**Review by karenkrieger — CHANGES_REQUESTED** [review]

**Review by JulianaMeyerArruda — COMMENTED** [review]
Commits feitos, após sugestões da @karenkrieger!

**Review by karenkrieger — APPROVED** [review]

**Pull request merged**